CONTRIBUTING.md

```diff
@@ -53,19 +91,30 @@ well. See [`test-suite/README.md`](test-suite/README.md) for how to do so.
 
-Don't be alarmed if the pull request process takes some time. It can take a few days to get feedback, approval on the final changes, and then a merge. Coq doesn't release new versions very frequently so it can take a few months for your change to land in a released version. That said, you can start using the latest Coq `master` branch to take advantage of all the new features, improvements, and fixes.
+Don't be alarmed if the pull request process takes some time. It can take a
+few days to get feedback, approval on the final changes, and then a merge.
+Do not hesitate to ping the reviewers if it takes longer than this.
+Coq doesn't release new versions very frequently so it can take a few months
+for your change to land in a released version. That said, you can start using
+the latest Coq `master` branch to take advantage of all the new features,
+improvements, and fixes.
 
-Whitespace discipline (do not indent using tabs, no trailing spaces, text files end with newlines) is checked by the `lint` job on GitLab CI (using `git diff --check`). We ship a [`dev/tools/pre-commit`](/dev/tools/pre-commit) git hook which fixes these errors at commit time. `configure` automatically sets you up to use it, unless you already have a hook at `.git/hooks/pre-commit`.
+Whitespace discipline (do not indent using tabs, no trailing spaces, text
+files end with newlines) is checked by the `lint` job on GitLab CI (using
+`git diff --check`). We ship a [`dev/tools/pre-commit`](/dev/tools/pre-commit)
+git hook which fixes these errors at commit time. `configure` automatically
+sets you up to use it, unless you already have a hook at `.git/hooks/pre-commit`.
 
-Here are a few tags Coq developers may add to your PR and what they mean. In general feedback and requests for you as the pull request author will be in the comments and tags are only used to organize pull requests.
+Here are a few tags Coq developers may add to your PR and what they mean. In
+general feedback and requests for you as the pull request author will be in
+the comments and tags are only used to organize pull requests.
 
 - [needs: rebase][rebase-label] indicates the PR should be rebased on top of
   the latest base branch (usually `master`). See the
   [GitHub documentation](https://help.github.com/articles/about-git-rebase/)
   for a brief introduction to using `git rebase`.
-  This label will be automatically added if you open or synchronize your PR and
-  it is not up-to-date with the base branch. So please, do not forget to rebase
-  your branch every time you update it.
+  We generally ask you to rebase only when there are merge conflicts or if
+  the PR has been opened for a long time and we want a fresh CI run.
 - [needs: fixing][fixing-label] indicates the PR needs a fix, as discussed in the comments.
 - [needs: benchmarking][benchmarking-label] and [needs: testing][testing-label]
   indicate the PR needs testing beyond what the test suite can handle.
```

Comment:
We have no section on `## Coding style`.

Maybe we can start a stub. I strongly support the convention that `ocamlformat` uses, so even if we don't check with the tool yet, we could provide an example configuration / guide.

Reply:
A few other things worth mentioning is that we discourage  using imperative style.

Reply:
Do you mean the style used by `ocamlformat` by default?
Unfortunately, explaining how to set it up wouldn't be very useful because if you want to apply it, you can only apply it to whole files at once.
We actually have this (quite outdated and completely unknown) document: https://github.com/coq/coq/blob/master/dev/doc/style.txt
and this outdated wiki page: https://github.com/coq/coq/wiki/CoqStyle

Reply:
Anyway, I propose to delay this to another PR because I'm not personally sure what to put.

Reply:
Good point, let's open an issue.

Reply:
https://github.com/coq/coq/issues/9572